Dockerfile

```diff
@@ -1,15 +1,7 @@
 ARG BASE_IMAGE=nvcr.io/nvidia/pytorch:24.03-py3
 
 FROM ${BASE_IMAGE}
-ARG APEX_TAG=59b80ee8df79cec125794949327f29913c328746
-ARG TE_TAG=7d576ed25266a17a7b651f2c12e8498f67e0baea
-ARG MLM_TAG=a3fe0c75df82218901fa2c3a7c9e389aa5f53182  # On: core_r0.8.0
-ARG NEMO_TAG=e033481e26e6ae32764d3e2b3f16afed00dc7218  # On: r2.0.0rc1
-ARG ALIGNER_COMMIT=main
 
-ARG PYTRITON_VERSION=0.5.10
-ARG PROTOBUF_VERSION=4.24.4
-ARG TRTLLM_VERSION=v0.10.0
```

Comment:
Actually we could revert this change. I studied the books again and cache invalidation happens only where ARGS are used, not where they are defined. So it would be fine to define all collectively at the top. I guess there are different tastes (some like to define it close to first usage, others as a "ToC" collectively). I'm fine with both, let me know what you'd prefer

Reply:
I'd prefer at the top. I think it's easier to inspect

Reply:
I double checked I could pull the image, so I think once that's reverted we can merge